config-linux.md

```diff
@@ -154,15 +154,20 @@ In addition to any devices configured with this setting, the runtime MUST also s
 ## Control groups
 
 Also known as cgroups, they are used to restrict resource usage for a container and handle device access.
-cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
+cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` is expected to be relative to the cgroups mount point.
-If `cgroupsPath` is not specified, implementations can define the default cgroup path.
+If `cgroupsPath` is:
+* ... an absolute path (starting with `/`), the implementation MUST take the path to be relative to the cgroup mount point.
+* ... a relative path (not starting with `/`), implementations SHOULD take the value to be the suffix of the resulting cgroup path.
+* ... not specified, implementations CAN define the default cgroup path.
+Implementations CAN consider certain `cgroupsPath` values to be invalid, and MUST generate an error if this is the case.
+If a `cgroupsPath` value is specified, an implementation MUST consistently attach to the same place in the cgroup hierarchy given the same value of `cgroupsPath`.
+
 Implementations of the Spec can choose to name cgroups in any manner.
 The Spec does not include naming schema for cgroups.
-The Spec does not support [split hierarchy][cgroup-v2].
+The Spec does not support the [unified or "default" hierarchy][cgroup-v2].
```

Comment:
The old wording is [“we don't support per-controller paths” for the reasons discussed in cgroup-v2.txt][1]. It's not about limiting the cgroup version.  Although you're now the [second person][2] I've heard interpret the old wording that way, so we probably want to adjust the wording.  I think we *do* want to support v2 cgroups though, and it may be more clear [than I thought][3] if the runtime joins both v1 and v2 cgroups as needed to collect the controllers it wants to join (because a given controller can only be exposed in the v1 or v2 hierarchy, and not in both at once).  So if you want to join the `devices` controller and that controller's in the v2 hierarchy, you join the appropriate v2 cgroup.

[1]: https://github.com/opencontainers/runtime-spec/pull/137#discussion_r38601118
[2]: https://github.com/opencontainers/ocitools/pull/93#discussion_r65658258
[3]: https://github.com/opencontainers/ocitools/pull/93#discussion_r65658513